sentry/src/main/java/io/sentry/CpuCollectionData.java

```diff
@@ -1,15 +1,25 @@
 package io.sentry;
 
 import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
 
 @ApiStatus.Internal
 public final class CpuCollectionData {
   final long timestampMillis;
   final double cpuUsagePercentage;
+  final @NotNull SentryDate timestamp;
```

Comment:
could we just have one (SentryDate) timestamp here and for memory/frames?

Reply:
we can change the collectors so that when `collect()` is called we can pass a timestamp, to have only one for cpu and memory
But frames are different, as they are captured with different intervals

Reply:
yeah I guess what I meant is `timestamp` already contains `timestampMillis` in it in a way, so we don't need to manage two values that are the same thing here

Reply:
done, thanks for the suggestion
another look would be great :D